Source/DafnyLanguageServer/README.md

```diff
@@ -121,4 +130,122 @@ Plugins are typically used to report additional diagnostics such as unsupported
 
 Note that all plugin errors should use the original program's expressions' token and NOT `Token.NoToken`, else no error will be displayed in the IDE.
 
-Morover, plugins should not write anything to `stdout` as it interferes with the communication protocol with the IDE.
```

Comment:
Is this no longer true?

Reply:
Yes, this is no longer true. Plugins can write to stdout now, and this information is recorded in a log file.

https://github.com/dafny-lang/dafny/blob/1472e40e4007588071b2bced71f753fb7b678f9b/Source/DafnyLanguageServer/Program.cs#L30-L32